app/build.gradle

```diff
@@ -60,7 +60,7 @@ android {
             buildConfigField "String", "MAPBOX_KEY", '"'+MAPBOX_KEY+'"'
             buildConfigField "String", "STRIPE_API_KEY", '"'+STRIPE_API_TOKEN+'"'
             buildConfigField "String", "PAYPAL_CLIENT_ID", '"'+PAYPAL_CLIENT_ID+'"'
-            resValue "string",  "FRONTEND_HOST", "eventyay.com"
+            resValue "string",  "FRONTEND_HOST", "api.eventyay.com"
```

Comment:
What does it mean?

Reply:
I told you to change the API URL for reproducing the issue, for debugging.

Reply:
Actually, Default base URL has "api.eventyay" and in the intent filter, the host had value "eventyay.com"  because of which options of choosing app was coming.

Reply:
I changed the API URL for reproducing the error. Do you want it to remain changed or back to initial value?

Reply:
You unfortunately, don't understand the issue. Read it again

Reply:
@iamareebjamal @liveHarshit If I am correct then the app deep linking feature is not working for the release build of the app and I am supposed to solve this issue.
Please correct me if  I am wrong?

Reply:
Yes, you're. App link intent should accept the only following type of URLs - 
1. Event details: https://eventyay.com/e/{event-identifier}
2. Reset password: https://eventyay.com/reset-password?token={token}
3. https://eventyay.com/email/verify?token={token}'

For other types like terms and policy URLs, it should no take as app link intent.

But here you removed the whole feature instead. We are not targeting any type of build. I told you to change frontend host only for testing purposes as all links are with eventyay.com host.
Clear??

Reply:
> I changed the API URL for reproducing the error. Do you want it to remain changed or back to initial value?

So we send PR after fixing the error, not after reproducing it.